.gitignore

```diff
@@ -17,6 +17,7 @@
 .env.development.local
 .env.test.local
 .env.production.local
+.env
```

Comment:
Where do you use it? If you use the `dotenv` library, please explain how to handle the variables in the README.